sys/sys/signal.h

```diff
@@ -385,6 +385,10 @@ struct sigaction {
 #define	NSIG		32	/* number of old signals (counting 0) */
 #endif
 
+#if __POSIX_VISIBLE >= 202405
+#define SIG2STR_MAX	32	/* size of buffer required for sig2str() */
```

Comment:
Why is this symbol in sys/signal.h?  It is not needed by the kernel, why cannot it be placed into include/signal.h?

Reply:
> Why is this symbol in sys/signal.h? It is not needed by the kernel, why cannot it be placed into include/signal.h?

I put it there because there are no SIGxxx definitions in include/signal.h but I can change it.

Reply:
Still not moved

Reply:
Fixed.